lib/solr/query/request.rb

```diff
@@ -20,13 +21,14 @@ module Solr
   module Query
     class Request
       attr_reader :search_term
-      attr_accessor :filters, :fields, :facets, :boosting, :debug_mode, :spellcheck,
-                    :limit_docs_by_field, :phrase_slop, :response_fields, :query_operator
+      attr_accessor :filters, :query_fields, :response_fields, :facets, :boosting, :debug_mode, :spellcheck,
```

Comment:
if we are going to use the standard solr naming I think it's a good idea to use `field_list` instead of `response_fields`.

Reply:
`field_list` might sound unclear for someone that is new to Solr, but I assume that most of our users will have some knowledge of solr, so I agree that `field_list` is better.